values-schema.yaml

```diff
@@ -1371,7 +1371,9 @@ properties:
           '^[a-z-]+$':
             $ref: '#/definitions/team'
   toolsVersion:
```

Comment:
remove this

Reply:
If I remove that the CI will fail. @j-zimnowoda can explain better why.

Reply:
@Morriz if you want `toolsVersion` to be part of values, then it has to be defined in values-schema.yaml

Reply:
nope, I said the opposite: remove it everywhere